### DISCUSSION TITLE
WindowsUIComposition will be invalid after NativeAOT is released.

### DISCUSSION THREAD

**Discussion by sudongg** [AvaloniaUI/Avalonia#8643] [General]
After I released it using the AvaloniaCoreRTDemo project (https://github.com/teobugslayer/AvaloniaCoreRTDemo), WindowsUIComposition doesn't work, I want to know if this problem can be fixed.

**Comment by maxkatz6** [comment]
Can you provide more details about your problem?
I don't remember any reason why winui comp shouldn't work with native aot.

**Reply by sudongg** [reply]
> Can you provide more details about your problem? I don't remember any reason why winui comp shouldn't work with native aot.

When NativeAOT is released, the rendering behavior is consistent with UseWindowsUIComposition = false. When UseWindowsUIComposition = false, when dragging the title bar, the animation will stop until the mouse is released, and the animation will refresh timing at 60 frames per second and not synchronized with the monitor.
